packages/fluentui/docs/src/examples/components/Button/Performance/ButtonMinimal.perf.tsx

```diff
@@ -1,7 +1,7 @@
 import { Button } from '@fluentui/react-northstar';
 import * as React from 'react';
 
-const ButtonMinimalPerf = () => <Button />;
+const ButtonMinimalPerf = () => <Button content="click" />;
```

Comment:
I don't that it's expected in this perf example as `content` will an additional slot

Reply:
![image](https://user-images.githubusercontent.com/14183168/79630251-0802f600-8150-11ea-8458-7a78b20413f8.png)

Reply:
Yeah, I understand there's a change on the screen, basically this:

<img width="929" alt="Screen Shot 2020-04-18 at 10 07 05 AM" src="https://user-images.githubusercontent.com/8545105/79631923-6635d600-815c-11ea-9b82-e858137eb887.png">
But otherwise it throws errors on console complaining about lack of label...

So I will change `label` for `aria-label="Minimal button"` so there's no visual changes and the error is gone... Not sure if it's proper so let me know if I should remove.